### PR TITLE
Rename command logging services

### PR DESCRIPTION
**Pull request by sroze**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 3.0 |
| Fixed tickets | ø |

This PR updates the name of the services used for the two listeners. The main reason is that if the developer simply copy/paste the two services' definition, it won't work as expected as they have the same name. The first one will be replaced.

### CODE REVIEW

File: cookbook/console/logging.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ First configure a listener for console exception events in the service container
 
         # app/config/services.yml
         services:
-            kernel.listener.command_dispatch:
```
Comment (javiereguiluz) [inline cookbook/console/logging.rst]: Unrelated to this pull request, but shouldn't these services be called `app.*` instead of `kernel.*` ?

Comment (sroze) [inline cookbook/console/logging.rst]: You are right, that makes more sense. Do you want me to update the PR?

Comment (xabbuh) [inline cookbook/console/logging.rst]: That would be good. There is no need to do this in another PR.

Comment (sroze) [inline cookbook/console/logging.rst]: @javiereguiluz @xabbuh updated.

**Comment by javiereguiluz** [comment]
@sroze 👍 nice! Thank you.

**Comment by xabbuh** [comment]
Thank you @sroze!
